design_docs/db/schema/content.yml

```
@@ -0,0 +1,60 @@
+# Schema for the MongoDB collection that stores content
+$schema: http://json-schema.org/draft-07/schema#
+type: object
+properties:
+  url:
```

```suggestion
properties:
  _id:
    bsonType: objectId
```

We should include the `_id` field in collection schemas

---

Also, we should consider using `bsonType` over `type` for these